tests/resources/main.tf

```diff
@@ -16,15 +28,61 @@ module "landing_zone" {
 }
 
 ##############################################################################
-# Observability Instances
+# COS instance
+##############################################################################
+
+module "cos" {
+  source            = "terraform-ibm-modules/cos/ibm"
+  version           = "8.11.14"
+  resource_group_id = module.resource_group.resource_group_id
+  cos_instance_name = "${var.prefix}-cos"
+  cos_tags          = var.resource_tags
+  create_cos_bucket = false
+}
+
+##############################################################################
+# COS buckets
+##############################################################################
+
+locals {
+  logs_bucket_name    = "${var.prefix}-logs-data"
+  metrics_bucket_name = "${var.prefix}-metrics-data"
+}
+
+module "buckets" {
+  source  = "terraform-ibm-modules/cos/ibm//modules/buckets"
+  version = "8.11.14"
+  bucket_configs = [
+    {
+      bucket_name            = local.logs_bucket_name
+      kms_encryption_enabled = false
+      region_location        = var.region
+      resource_instance_id   = module.cos.cos_instance_id
+    },
+    {
+      bucket_name            = local.metrics_bucket_name
+      kms_encryption_enabled = false
+      region_location        = var.region
+      resource_instance_id   = module.cos.cos_instance_id
+    }
+  ]
+}
+
+##############################################################################
+# Observability:
+# - Log Analysis instance
```

Comment:
do not deploy a Log Analysis instance - we are not going to deploy those agents by default anymore, and neither should the tests